docs/tutorials/mastering.md

```diff
@@ -85,21 +93,7 @@ Install LXD:
 sudo snap install lxd
 ```
 
-Create custom storage pool and LXD project:
-
-```console
-lxc storage create sdcore zfs size=150GiB
-lxc project create sdcore --config features.images=true --config features.networks=true --config features.networks.zones=true --config features.profiles=true --config features.storage.buckets=true --config features.storage.volumes=true
-```
-
-```{note}
-When using LXD version lower than 5.9, the command above will produce the following error:
-> Error: Invalid project configuration key "features.networks.zones"
-
-Please upgrade your LXD to the latest version or remove unsupported config from the command.
-```
-
-Create the configuration to initialize the LXD.
+Create LXD configuration file:
```

Comment:
I realize the LXD installation is not part of the changes for this tutorial, but as 5.0.2 is the default on new 22.04 VMs, I am thinking we need to update the `snap install lxd` to make sure we use the correct version

Reply:
```
sudo snap install lxd && sudo snap refresh lxd --channel=latest
```

Reply:
We've originally decided to add a note:
```
When using LXD version lower than 5.9, the command above will produce the following error:
> Error: Invalid project configuration key "features.networks.zones"

Please upgrade your LXD to the latest version or remove unsupported config from the command.
```
Just in case someone wants to use older version of LXD. Not sure if that makes sense, but at least we give the user the choice. Do you think we should always require using latest version?

Reply:
the init command fails otherwise

Reply:
Yes. And the note and the comment inside the `preseed.yaml` instructs the user to remove unsupported key `features.networks.zones` from the config if you wanna go with an older version.

Reply:
Then why don't we remove that by default?

Reply:
What I mean is by default, this is going to fail unless the user specifically went out of their way to upgrade LXD prior to running the tutorial

Reply:
Of course :D now that you've mentioned it, it seems so obvious :D Thank you!
Removed.

Reply:
grrr. Now I'm getting
```
Error: Failed to create local member project "sdcore": Invalid project configuration key "features.storage.buckets"
```